js/src/Components/Widgets/LinkWidget.js

```diff
@@ -1,20 +1,44 @@
 import React, {useState} from 'react';
 import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
 import FormGroup from '@material-ui/core/FormGroup';
 import FormControl from '@material-ui/core/FormControl';
 import FormLabel from '@material-ui/core/FormLabel';
 import {FormHelperText} from "@material-ui/core";
 import Autocomplete from '@material-ui/lab/Autocomplete';
 
 export const LinkWidget = ({fieldId, defaultValue, onFieldChange, settings}) => {
-
   let timeout;
+  let initialCondition = defaultValue;
+  if (initialCondition === undefined){
+    initialCondition = [];
+    initialCondition.push(emptyLinkValue());
+  }
+
   const [urlSuggestions, setSuggestions] = useState([]);
+  const [fieldValues, setValues] = useState(initialCondition);
 
-  const defaultFieldValue = {
-    uri: defaultValue && defaultValue.length ? defaultValue[0].uri : '',
-    title: defaultValue && defaultValue.length ? defaultValue[0].title : ''
-  };
+  function emptyLinkValue() {
+    return {
+      uri: '',
+      title: '',
+      options: [],
+      target_uuid: false
+    };
+  }
+
+  const alterValues = (props) => {
```

Comment:
`props` is a common variable in stateful components. the variable here should be named something different.

Reply:
fixed.